### PR TITLE
Add test case to verify the serialization working correctly

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
RN, we don't have a test case to verify the serialization and deserialization are working correctly.

## What is the new behavior?
Added a test case to verify the serialization and deserialization

## Does this introduce a breaking change?
Nope

### CODE REVIEW

File: python-sdk/tests_integration/sql/operators/test_load_file.py [hunk 1]
```diff
@@ -1138,3 +1139,25 @@ def test_aql_nested_ndjson_file_to_database_explicit_illegal_sep_params(sample_d
...
+        def serialize():
+            return init_table
+
+        @task(task_id="verify")
+        def verify(t):
+            assert t == init_table
+
+        t = serialize()
+        verify(t)
+    test_utils.run_dag(sample_dag)
```
Comment (kaxil) [inline python-sdk/tests_integration/sql/operators/test_load_file.py:1163]: Does not have to been an integration test for sure!

File: python-sdk/tests_integration/sql/operators/test_load_file.py [hunk 2]
```diff
@@ -1138,3 +1139,25 @@ def test_aql_nested_ndjson_file_to_database_explicit_illegal_sep_params(sample_d
     df = db.export_table_to_pandas_dataframe(test_table)
     assert df.shape == (1, 36)
     assert "payload_size" in df.columns
+
+
+def test_serialization_and_deserialization_of_table(sample_dag):
+    """
+    Test serialization and deserialization of table object
+    """
```
Comment (kaxil) [inline python-sdk/tests_integration/sql/operators/test_load_file.py:1147]: Doesn't add any value, says the same thing as the test name

File: python-sdk/tests_integration/sql/operators/test_load_file.py (hunk 1 above)
Comment (dimberman) [inline python-sdk/tests_integration/sql/operators/test_load_file.py:1163]: @utkarsharma2 aas Kaxil mentioned. Ultimately we are just serializing data into json and then deserializing it from json. There is no need to create a DAG run and if anything that adds unnecessary overhead. Why not just serialize it into a json variable and ensure that the resulting json looks correct/can deserialize back to the same object?

**Review by dimberman — CHANGES_REQUESTED** [review]

**Comment by utkarsharma2** [comment]
No longer needed. Closing the PR.

**Pull request closed without merging**